main.py

```
@@ -1,17 +1,26 @@
     # TODO: create the logging hierarchy and pass it to the modules.
-    view = View()
-    controller = Controller(view=view, db_name="model/telemetry.db")
+    view = View(passenger_number_config)
+    controller = Controller(view=view, db_name=f"model/{db_name_config}",
+                            server_ip_config=server_ip_config,
+                            passenger_number_config=passenger_number_config)
     try:
         view.start_mainloop()
     except KeyboardInterrupt:
         controller.close_on_escape()
+    except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)